python/hail/dataset.py

```diff
@@ -2142,9 +2142,10 @@ def ibd_prune(self, threshold, tiebreaking_expr=None, maf=None, bounded=True):
 
         **Examples**
 
-        Prune samples so that no two have a PI_HAT value greater than or equal to 0.5:
+        Prune samples so that no two have a PI_HAT value greater than or equal to 0.5, with a tiebreaking expression that 
+        selects cases over controls:
 
-        >>> pruned_vds = vds.ibd_prune(0.5)
+        >>> pruned_vds = vds.ibd_prune(0.5, "if (sa1.isCase) 1 else 0")
```

Comment:
I think you should split this out into two separate examples. Just the basic one, and the one that uses the tiebreaking expression. Also, use `tiebreaking_expr='...'` since it has a default